build/yaml/samples/named-port-sample/rule-out-rule-applied-to-with-dst-with-dup-port-multi.yml

```diff
@@ -0,0 +1,108 @@
+# DB
+# first run
+#  kubectl label ns db2 role=db
+#  test: kubectl exec -ti web-ip -n web -- nc db-ip 3306 -v
+#  kubectl exec -ti web-ip -n web -- nc db-ip 1234 -v
+#  nc db-ip 1234 -v
```

Comment:
Please polish the description, remove meaningless words and unify the formats.
The same issues in the other yamls.